--- src/page_export_avi.cc.orig	2007-03-17 12:26:16.000000000 +0100
+++ src/page_export_avi.cc	2007-03-17 12:30:18.000000000 +0100
@@ -211,8 +211,10 @@
 		time_t datetime = time( NULL );
 		int frameNum = 0;
 		int i;
-		bool resample = gtk_toggle_button_get_active( GTK_TOGGLE_BUTTON(
-		                    lookup_widget( common->getWidget(), "checkbutton_export_avi_resample" ) ) );
+#if BYTE_ORDER == LITTLE_ENDIAN
+                bool resample = gtk_toggle_button_get_active( GTK_TOGGLE_BUTTON(
+                                    lookup_widget( common->getWidget(), "checkbutton_export_avi_resample" ) ) );
+#endif
 		int16_le_t *audio_buffers[ 4 ];
 		AudioInfo info;
 		AsyncAudioResample<int16_ne_t,int16_le_t>* resampler = 0;
