--- kino-0.8.1/Kino.desktop~	2006-05-02 20:47:49.000000000 +0300
+++ kino-0.8.1/Kino.desktop	2006-05-02 20:49:15.000000000 +0300
@@ -12,6 +12,7 @@
 GenericName[hu]=Videoszerkesztő
 GenericName[it]=Editor Video
 GenericName[sv]=Videoredigering
+GenericName[pl]=Edytor video
 GenericName[nb]=Videoredigering
 GenericName[no]=Videoredigerer
 GenericName[ru]=Видеоредактор
@@ -27,6 +28,7 @@
 Comment[hu]=DV videók szerkesztése
 Comment[it]=Montaggio Video DV non lineare
 Comment[sv]=Redigera DV video
+Comment[pl]=Narzędzie do edycji DV
 Comment[nb]=Redigering av DV video
 Comment[no]=Redigere DV video
 Comment[ru]=Редактирование DV видео
@@ -37,5 +39,5 @@
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=Application;AudioVideo;
+Categories=GTK;GNOME;AudioVideo;AudioVideoEditing;
 MimeType=video/dv;application/x-smil
