--- Kino.desktop	2009-03-28 17:54:22.000000000 +0100
+++ Kino.desktop.new	2010-01-28 23:43:53.934366935 +0100
@@ -16,6 +16,7 @@
 GenericName[sv]=Videoredigering
 GenericName[nb]=Videoredigering
 GenericName[no]=Videoredigerer
+GenericName[pl]=Edytor video
 GenericName[ru]=Видеоредактор
 GenericName[tt]=Видеоредактор
 GenericName[uk]=Відеоредактор
@@ -35,6 +36,7 @@
 Comment[sv]=Redigera DV video
 Comment[nb]=Redigering av DV video
 Comment[no]=Redigere DV video
+Comment[pl]=Narzędzie do edycji DV
 Comment[ru]=Редактирование DV-видео
 Comment[tt]=DV видео редакторлау
 Comment[uk]=Редагування DV відео
@@ -45,5 +47,5 @@
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=Application;AudioVideo;AudioVideoEditing;
+Categories=GTK;GNOME;AudioVideo;AudioVideoEditing;
 MimeType=video/dv;application/smil
