--- Kino.desktop.orig	2007-02-22 08:35:13.000000000 +0100
+++ Kino.desktop	2007-03-17 12:19:09.000000000 +0100
@@ -8,10 +8,11 @@
 GenericName[el]=Επεξεργαστής Βίντεο
 GenericName[es]=Editor de vídeo
 GenericName[eu]=bideo-editorea
-GenericName[fr]=Montage vido
+GenericName[fr]=Montage vidéo
 GenericName[hu]=Videoszerkesztő
 GenericName[it]=Editor Video
 GenericName[sv]=Videoredigering
+GenericName[pl]=Edytor video
 GenericName[nb]=Videoredigering
 GenericName[no]=Videoredigerer
 GenericName[ru]=Видеоредактор
@@ -23,10 +24,11 @@
 Comment[el]=Επεξεργαστής Βίντεο
 Comment[es]=Editar vídeos DV
 Comment[eu]=Editatu DV bideoak
-Comment[fr]=Montage vido DV
+Comment[fr]=Montage vidéo DV
 Comment[hu]=DV videók szerkesztése
 Comment[it]=Montaggio Video DV non lineare
 Comment[sv]=Redigera DV video
+Comment[pl]=Narzędzie do edycji DV
 Comment[nb]=Redigering av DV video
 Comment[no]=Redigere DV video
 Comment[ru]=Редактирование DV видео
@@ -37,5 +39,5 @@
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=Application;AudioVideo;
+Categories=GTK;GNOME;AudioVideo;AudioVideoEditing;
 MimeType=video/dv;application/smil
